app/mailers/vote_mailer.rb

```diff
@@ -0,0 +1,15 @@
+class VoteMailer < ApplicationMailer
+  default from: 'notifications@example.com'
+
+  def vote_email(review)
+    @review = review
+    @user = review.user
+    mail(to: @user.email, subject: 'Post Upvoted')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.